src/components/ModalDelete.js

```diff
@@ -0,0 +1,28 @@
+import Modal from "./Modal.js";
+
+export default class ModalDelete extends Modal {
+  constructor({ modalSelector, cardRemovalHandler }, card) {
+    super({ modalSelector });
+    this._deleteButton = document.getElementById("delete-image-confirm");
+    this._cardRemovalHandler = cardRemovalHandler;
+    this.card = card;
+  }
+  open() {
+    this._setEventListeners();
```

Comment:
`this._setEventListeners();`


Please, delete it


`this.setEventListeners()` should be removed from `open` because it's a **public** method that should be called only **once** in `index.js` otherwise you add listeners again and again to the same elements. This can cause a **`memory leak`**